internal/test/test.go

```diff
@@ -617,3 +617,10 @@ func (h *Helper) GetCommit(repo string) string {
 	}
 	return strings.TrimSpace(string(out))
 }
+
+func (h *Helper) RemoveFile(path string) {
+	err := os.RemoveAll(path)
```

Comment:
No, it's for removing everything, file, directories with files in them. This this PR, it's for deleting a directory. Maybe `RemoveDir()` would be a better name for the function.